bodhi/server/templates/update.html

```diff
@@ -219,8 +219,9 @@ <h4>How to install</h4>
                   <div class="col pr-0">
                     <%
                       help_content = "Comments support Fedora-Flavored Markdown"
+                      privacy_link = request.registry.settings.get('privacy_link')
```

Comment:
I think it's better to assign the variable only after having checked `request.registry.settings.get('privacy_link')` is set. I would move this line inside the if statement below.